spec.md

```diff
@@ -1617,6 +1626,40 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 |-----------|-----------|-------------|-------------------|
 | Volume does not exist | 5 NOT_FOUND | Indicates that a volume corresponding to the specified `volume_id` does not exist. | Caller MUST verify that the `volume_id` is correct and that the volume is accessible and has not been deleted before retrying with exponential back off. |
 
+
+#### `ModifyVolume`
+
+A Controller plugin SHALL implement this RPC call. If the MODIFY_VOLUME capability is present, a Controller plugin MUST implement this RPC call. This RPC allows the CO to change mutable key attributes of a volume. 
+
+This operation MUST be idempotent. The preset parameters in the VolumePerformanceClass can be greater than or less than the current values. 
```

Comment:
VolumePerformanceClass is a K8s term so it should not be in the CSI spec.
Suggest changing "The preset parameters in the VolumePerformanceClass can be greater than or less than the current values." to "The new mutable parameters in ModifyVolume can be different from the existing mutable parameters".